tasks/publish.yaml

```diff
@@ -57,22 +55,24 @@ tasks:
       - description: "Publish the bundles"
         cmd: |
           set -e
-          uds publish bundles/k3d-standard/uds-bundle-k3d-core-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
-          uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
 
-          # dont publish arm64 for registry1 since IB images are only amd64
+
+          # dont publish dev & demo bundles for registry1 flavor
           if [ "${FLAVOR}" != "registry1" ]; then
-            uds publish bundles/k3d-standard/uds-bundle-k3d-core-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
-            uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
+            uds publish bundles/k3d-standard/uds-bundle-k3d-core-demo-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
+            uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-dev-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
+            uds publish bundles/k3d-standard/uds-bundle-k3d-core-demo-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
+            uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-dev-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
           fi
 
       - description: "Tag the latest bundles"
         cmd: |
           set -e
-          pkgPath="ghcr.io/defenseunicorns/packages/uds/bundles/${FLAVOR}/k3d-core"
+          pkgPath="ghcr.io/defenseunicorns/packages/uds/bundles/k3d-core-demo"
           # dont tag arm64 for registry1 since IB images are only amd64
           if [ "${FLAVOR}" != "registry1" ]; then
             zarf tools registry copy ${pkgPath}:${VERSION}-arm64 ${pkgPath}:arm64
+            zarf tools registry copy ${pkgPath}:${VERSION}-amd64 ${pkgPath}:amd64
           fi
 
```

Comment:
nit: same comment here, can drop the flavor check. Also lint caught the trailing space here.